packages/hub/plugin-loader.js

```diff
@@ -276,6 +278,10 @@ class ConfiguredPlugins {
         copied.attributes.enabled = true;
       }
       this._plugins[copied.id] = copied;
+
+      if (copied.attributes['is-root-plugin']) {
+        this.rootPlugin = copied;
```

Comment:
This is the only remaining place I see `is-root-plugin`, is this one a leftover?

Reply:
Oops! Yeah that is unnecessary now. I’ll clean it up.